spec/collator.html

```diff
@@ -49,46 +49,39 @@
 
       <emu-alg>
         1. If collator has an [[initializedIntlObject]] internal slot with value *true*, throw a *TypeError* exception.
```

Comment:
good catch, will fix it in another commit. (I suspect that there are few more like this one, and I would love to have a emu validator to catch them all). @bterlson can we get a linter? jajajaja